src/rdf4cpp/rdf/datatypes/owl/Rational.cpp

```diff
@@ -2,6 +2,15 @@
 
 namespace rdf4cpp::rdf::datatypes::registry {
 
+static owl::Rational::cpp_type simplify(owl::Rational::cpp_type const &r) noexcept {
+    auto const divisor = gcd(numerator(r), denominator(r));
```

Comment:
This looks strange. Given r = 2/4 (=.5); divisor would be gcd(2,4) = 2- so 2/4 simplified would be 2/4/2 (= .25). 

Or am I overlooking sth?